### ISSUE TITLE
after following instructions theme doesn't change how to debug this pls help

### ISSUE THREAD

**Issue by markmark1** [dexpota/kitty-themes#51]
git clone --depth 1 git@github.com:dexpota/kitty-themes.git ~/.config/kitty/kitty-themes
cd ~/.config/kitty
ln -s ./kitty-themes/themes/DimmedMonokai.conf ~/.config/kitty/theme.conf

Add this line to your kitty.conf configuration file:
include ./theme.conf

these are steps i followed 

after following instructions theme doesn't change how to debug this pls help

**Comment by manojuppala** [comment]
i am having the same issue. the theme dosent change even after executing the following commands:
`
THEME=https://raw.githubusercontent.com/dexpota/kitty-themes/master/themes/gruvbox_dark.conf
`
`
wget "$THEME" -P ~/.config/kitty/kitty-themes/themes
`

**Comment by manojuppala** [comment]
@xiahuaS yes that worked. however i also had to comment out `#foreground`, `#cursor #cccccc` and all the colors for example
`#color1 #cc0403`, `#color9 #f2201f`.

**Comment by en1tan** [comment]
I commented out all of this and still it did not work

**Comment by dexpota** [comment]
I have just tested the commands and they work, you need obviously to restart `kitty` to see the changes.

It should not be necessary to comment out anything from the theme, if you change these settings you will end up with a different custom theme.

@markmark1 If you please can give some other information, are any of these commands giving you an error?

**Comment by dexpota** [comment]
@manojuppala I tested the commands and they work. Can you elaborate more? do you get any errors? Maybe the symbolic link already axists.
